src/components/elements/MoralisLogo/MoralisLogo.tsx

```diff
@@ -6,7 +6,7 @@ const MoralisLogo = () => {
 
   return (
     <Image
-      src={colorMode === 'dark' ? '/Moralis-DarkBG.svg' : '/Moralis-LightBG.svg'}
+      src={colorMode === 'dark' ? '/daoad.png' : '/Moralis-LightBG.svg'}
```

Comment:
Instead of repurposing `MoralisLogo` component to output a non-Moralis logo, it might be better to create a new Logo Component, and call on the component directly from the page that requires it.